cmd/bitxhub/client/validators.go

```diff
@@ -0,0 +1,38 @@
+package client
+
+import (
+	"fmt"
+
+	"github.com/hokaccha/go-prettyjson"
+	"github.com/meshplus/bitxhub/internal/repo"
+	"github.com/urfave/cli"
+)
+
+func validatorsCMD() cli.Command {
+	return cli.Command{
+		Name:   "validators",
+		Usage:  "Query validator address",
+		Action: getValidators,
+	}
+}
+
+func getValidators(ctx *cli.Context) error {
```

Comment:
`cmd` should get validators information by grpc gateway.
Ref: https://github.com/meshplus/bitxhub/blob/master/cmd/bitxhub/client/chain.go